proposal/more-math-functions.md

```diff
@@ -0,0 +1,445 @@
+  * Return NaN if `$base` < 0 and `$exponent` is not an integer.
+
+  * Return Infinity if `$base` is 0 and `$exponent` < 0, or if `$base` is
+    Infinity and `$exponent` > 0.
+  * If `$base` is -0 and `$exponent` < 0, or if `$base` is -Infinity and
+    `$exponent` > 0:
+    * Return -Infinity if `$exponent` is an odd integer.
+    * Return Infinity.
+
+  * Return 0 if `$base` is 0 and `$exponent` > 0, or if `$base` is Infinity and
+    `$exponent` < 0.
+  * If `$base` is -0 and `$exponent` > 0, or if `$base` is -Infinity and
+    `$exponent` < 0:
+    * Return -0 if `$exponent` is an odd integer.
+    * Return 0.
+
+  * Return `$base` raised to the power of `$exponent`.
+
+#### `sqrt()`
+
+```
+sqrt($number)
+```
+
+* Throw error if `$number` has units.
+* Return NaN if `$number` < 0.
+* Return -0 if `$number` is -0.
+* Return Infinity if `$number` is infinite.
+* Return the square root of `$number`.
+
+### Trigonometry
+
+#### Procedures
+
+##### Verify Inputs
+
+`cos($number), sin($number), and tan($number)` throw error if `$number` has
+units but is not a rotation value. If `$number` is unitless, it is assumed to be
+in radians.
+
+`acos($number), asin($number), and atan($number)` throw error if `$number` has
+units.
+
+Both inputs to `atan2($y, $x)` must have the same unit or be unitless. If not,
+throw error.
+
+#### `cos()`
+
+```
+cos($number)
+```
+
+* [Verify inputs](#verify-inputs).
+* Return NaN if `$number` is infinite.
+* Return cos(`$number`).
+
+#### `sin()`
+
+```
+sin($number)
+```
+
+* [Verify inputs](#verify-inputs).
+* Return NaN if `$number` is infinite.
+* Return -0 if `$number` is -0.
+* Return sin(`$number`).
+
+#### `tan()`
+
+```
+tan($number)
+```
+
+* [Verify inputs](#verify-inputs).
+* Return NaN if `$number` is infinite.
+* Return -0 if `$number` is -0.
+* If `$number` is an asymptote value:
+  * Return Infinity if `$number` is 90deg +/- 360*n* deg, where *n* is any integer.
+  * Return -Infinity if `$number` is -90deg +/- 360*n* deg, where *n* is any
+    integer.
+* Return tan(`$number`).
+
+#### `acos()`
+
+```
+acos($number)
+```
+
+* [Verify inputs](#verify-inputs).
+* Return NaN if [abs](#abs)(`$number`) > 1.
+* Return 0 if `$number` is 1.
+* Return acos(`$number`).
+
+#### `asin()`
+
+```
+asin($number)
+```
+
+* [Verify inputs](#verify-inputs).
+* Return NaN if [abs](#abs)(`$number`) > 1.
+* Return -0 if `$number` is -0.
+* Return asin(`$number`).
+
+#### `atan()`
+
+```
+atan($number)
+```
+
+* [Verify inputs](#verify-inputs).
+* Return -0 if `$number` is -0.
+* Return -90deg if `$number` is -Infinity.
+* Return 90deg if `$number` is Infinity.
+* Return atan(`$number`).
+
+#### `atan2()`
+
+`atan2($y, $x)` is distinct from `atan($y / $x)` because it preserves the
+quadrant of the point in question. For example, `atan2(1, -1)` corresponds to
+the point (-1, 1), which returns 135deg. In contrast, `atan(1 / -1)` and
+`atan(-1 / 1)` resolve first to `atan(-1)`, so both return -45deg.
```

Comment:
As a convention, we write non-normative text in block quotes:

```markdown
> `atan2($y, $x)` is distinct from `atan($y / $x)` because it preserves the
> quadrant of the point in question. For example, `atan2(1, -1)` corresponds to
> the point (-1, 1), which returns 135deg. In contrast, `atan(1 / -1)` and
> `atan(-1 / 1)` resolve first to `atan(-1)`, so both return -45deg.
```

Reply:
Good to know, thanks